doc/lua_api.txt

```diff
@@ -5186,9 +5186,10 @@ Sounds
 * `minetest.sound_fade(handle, step, gain)`
     * `handle` is a handle returned by `minetest.sound_play`
     * `step` determines how fast a sound will fade.
-      Negative step will lower the sound volume, positive step will increase
-      the sound volume.
+      The gain will change by this much per second,
+      until it reaches the target gain.
```

Comment:
should have a note along the lines of
"Note that prior to Minetest 5.4.0 the sign of `step` had to be negative to fade to a lower volume"